Fix mwe2 version in pom.xml

Make sure we use version 2.9.0 to avoid such building errors:

[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.4.0:java (mwe2Launcher) on project org.example.entities: Execution mwe2Launcher of goal org.codehaus.mojo:exec-maven-plugin:1.4.0:java failed: Plugin org.codehaus.mojo:exec-maven-plugin:1.4.0 or one of its dependencies could not be resolved: Failed to collect dependencies for org.codehaus.mojo:exec-maven-plugin:jar:1.4.0 (): Could not resolve version conflict among [org.codehaus.mojo:exec-maven-plugin:jar:1.4.0 -> org.eclipse.emf:org.eclipse.emf.mwe2.launch:jar:[2.9.0,2.10) -> org.eclipse.emf:org.eclipse.emf.mwe2.runtime:jar:[2.9.1.201705291010,2.9.1.201705291010], org.codehaus.mojo:exec-maven-plugin:jar:1.4.0 -> org.eclipse.emf:org.eclipse.emf.mwe2.launch:jar:[2.9.0,2.10) -> org.eclipse.emf:org.eclipse.emf.mwe2.language:jar:[2.9.1.201705291010,2.9.1.201705291010] -> org.eclipse.emf:org.eclipse.emf.mwe2.runtime:jar:[2.9.1.201705291010,2.9.1.201705291010], org.codehaus.mojo:exec-maven-plugin:jar:1.4.0 -> org.eclipse.xtext:org.eclipse.xtext.xtext.generator:jar:2.11.0 -> org.eclipse.emf:org.eclipse.emf.mwe.utils:jar:1.3.20.201605261059 -> org.eclipse.emf:org.eclipse.emf.mwe.core:jar:[1.3.20.201605261059,1.3.20.201605261059] -> org.eclipse.emf:org.eclipse.emf.mwe2.runtime:jar:[2.9.0.201605261059,2.9.0.201605261059], org.codehaus.mojo:exec-maven-plugin:jar:1.4.0 -> org.eclipse.xtext:org.eclipse.xtext.xtext.generator:jar:2.11.0 -> org.eclipse.emf:org.eclipse.emf.mwe2.lib:jar:2.9.0.201605261059 -> org.eclipse.emf:org.eclipse.emf.mwe2.runtime:jar:[2.9.0.201605261059,2.9.0.201605261059]] -> [Help 1]

--- a/org.example.hellomaven.parent/org.example.hellomaven/pom.xml
+++ b/org.example.hellomaven.parent/org.example.hellomaven/pom.xml
@@ -39,7 +39,7 @@
 					<dependency>
 						<groupId>org.eclipse.emf</groupId>
 						<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
-						<version>[2.9.0,2.10)</version>
+						<version>2.9.0.201605261059</version>
 					</dependency>
 					<dependency>
 						<groupId>org.eclipse.xtext</groupId>

--- a/org.example.helloweb.parent/org.example.helloweb/pom.xml
+++ b/org.example.helloweb.parent/org.example.helloweb/pom.xml
@@ -39,7 +39,7 @@
 					<dependency>
 						<groupId>org.eclipse.emf</groupId>
 						<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
-						<version>[2.9.0,2.10)</version>
+						<version>2.9.0.201605261059</version>
 					</dependency>
 					<dependency>
 						<groupId>org.eclipse.xtext</groupId>

--- a/org.example.parent/pom.xml
+++ b/org.example.parent/pom.xml
@@ -9,7 +9,7 @@
 	<properties>
 		<tycho-version>0.25.0</tycho-version>
 		<xtextVersion>2.11.0</xtextVersion>
-		<mwe2Version>[2.9.0,2.10)</mwe2Version>
+		<mwe2Version>2.9.0.201605261059</mwe2Version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>

